### ISSUE TITLE
Roman name field gone...in old entries.

### ISSUE THREAD

**Issue by JSBLGPN** [eXistSolutions/LGPN#115]
e.g. Aur. Hestiaios : Εστιαιος from Sidon

**Comment by tuurma** [comment]
case discussed is http://admin.exist-db.org:41233/exist/apps/lgpn/editor.xhtml?id=P5c4a30ad-7663-4d50-b22f-10f10ce00536

@JSBLGPN do you mean that the Roman Name section is not present and there are no means to reinstate it? Like in #102?

**Comment by tuurma** [comment]
```
for $i in collection('/db/apps/lgpn-data/data/persons')//tei:person[not(.//tei:nym[@type='Roman'])]
return $i
```

281 cases like this found at the moment

**Comment by JSBLGPN** [comment]
Yes

**Comment by tuurma** [comment]
Thanks for confirmation. In that case it will be fixed with #100, hopefully later today.
